modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -2112,11 +2112,15 @@ parameters should be positive.
 Must fall between 0 and max_theta.
 @param max_theta For standard and multi-scale Hough transform, maximum angle to check for lines.
 Must fall between min_theta and CV_PI.
+@param use_weight True if you want to use weighted Hough transform.
+@param thre_edgeval For weighted Hough transform, only account for over this
```

Comment:
No such params.

Reply:
I'm sorry, it was my mistake. I updated it.